Bahatiroben/MyEvents
# implementing google authentication

- user can sign in using their google email account

#### What does this PR do?
- it enables users to signup or login using a google account
#### Description of Task to be completed?
- as a user, I should be able to sign in using my google email account
#### How should this be manually tested?
- clone the repo
- set environment variables as specified in `.env.example`
- go to the browser and navigate to `http://localhost:3000/api/v1/users/auth/google`
- you should be able to log in and receive a response
#### Any background context you want to provide?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/39488016/67234359-45a6f080-f445-11e9-981b-17e3657f715f.png)

#### Questions:
